extensions/test/algorithms/dissolve.cpp

```diff
@@ -652,7 +651,9 @@ void test_all(ut_settings const& settings_for_sensitive_cases)
     TEST_DISSOLVE_WITH(dissolve_reallife, 91756.916526794434, 1, 0, 25,
                        settings_for_sensitive_cases);
 
+#if defined(BOOST_GEOMETRY_TEST_FAILURES)
     TEST_DISSOLVE(gitter_2013_04_a, 3043.9181, 3, 0, 21);
+#endif
```

Comment:
Is this a regression in dissolve introduced by this PR?

Reply:
Not that I know of. It did not compile before, so I did not bisect when it started failing